demos/interactive_face_detection_demo/gapi/README.md

```diff
@@ -9,7 +9,7 @@ This demo executes the Face Detection, Age/Gender Recognition, Head Pose Estimat
 * `emotions-recognition-retail-0003`, which is executed on top of the results of the first model and reports an emotion for each detected face
```

Comment:
I think the directory this demo is in should be `cpp_gapi`, rather than plain `gapi`, because "C++" and "G-API" aren't mutually exclusive.

Reply:
Renamed